packages/uma-auth-client/src/hooks/nwc-requests/useGetInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { type GetInfoResponse } from "src/Nip47Types";
+import { type NwcRequester } from "src/NwcRequester";
+import { useNwcRequester } from "../useNwcRequester";
+import { useOAuth } from "../useOAuth";
+
+export const useGetInfo = () => {
```

Comment:
I kind of wonder if there's a way for us to make these generic to have one that works for all requests with a request and response type param. They're all pretty repetitive.

Reply:
Yeah that's a good point; should be doable with generics? Will look into later